doc-content/enterprise-only/installation/securing-passwords-jws-proc.adoc

```diff
@@ -0,0 +1,133 @@
+[id='securing-passwords-jws-proc_{context}']
+
+= Securing passwords with a keystore
+
+You can use a keystore to encrypt passwords that are used for communication between {CENTRAL} and {KIE_SERVER}. You should encrypt both controller and {KIE_SERVER} passwords. If {CENTRAL} and {KIE_SERVER} are deployed to different application servers then both application servers should use the keystore.
+
+Use Java Cryptography Extension KeyStore (JCEKS) for your keystore because it supports symmetric keys. Use KeyTool, which is part of the JDK installation, to create a new JCEKS.
+
+[NOTE]
+====
+If {KIE_SERVER} is not configured with JCEKS, {KIE_SERVER} passwords are stored in system properties in plain text form.
+====
+
+.Prerequisites
+
+* {KIE_SERVER} is installed in
+ifeval::["{context}" == "install-on-jws"]
+{JWS}.
+endif::[]
+ifeval::["{context}" == "install-on-tomcat"]
+{TOMCAT}.
+endif::[]
+* Java 8 or higher is installed.
+
+.Procedure
+ifeval::["{context}" == "install-on-jws"]
+. Open the `_TOMCAT_HOME_/tomcat/conf/tomcat-users.xml` file in a text editor.
```

Comment:
Throughout this file you use _TOMCAT_HOME_ etc. and then _JWS_HOME/tomcat_. This is the JWS version of the file so I assume that's a mistake?

Reply:
I believe this is correct but I'll double check with Karel.